docs/aws/ContinuousDeployment.rst

```diff
@@ -13,16 +13,20 @@ You can enable continuous deployment of the changes in your source repository to
 
    It is optional to keep the deployment in your account automatically synchronized with the source code repository.
 
-For the continuous deployment to work, you need to fork the source code to register the webhook listener that triggers an upgrade of your deployment.
+For the continuous deployment to work, complete the following steps:
 
-After forking, make sure to update the ``repository.url`` in the :file:`package.json` file in your fork.
+1. Fork the source code to register the webhook listener that triggers an upgrade of your deployment.
```

Comment:
This also changes the meaning of the documentation.
It is not correct that forking the source code will register the webhook listener.

Reply:
I see, it is in the original documentation like this.

Reply:
Made a note in #129